# std_msgs/Header header
string targetPoint
---
bool finishPublishCommand